policyengine_us/variables/gov/states/nj/tax/income/credits/eitc/nj_childless_eitc_age_eligible.py

```diff
@@ -0,0 +1,37 @@
+from policyengine_us.model_api import *
+
+
+class nj_childless_eitc_age_eligible(Variable):
+    value_type = bool
+    entity = TaxUnit
+    label = "New Jersey Eligible for EITC"
+    definition_period = YEAR
+    reference = "https://law.justia.com/codes/new-jersey/2022/title-54a/section-54a-4-7/"
+    defined_for = StateCode.NJ
+
+    def formula(tax_unit, period, parameters):
+        # Return True if all federal EITC conditions are met, except with modified age paramaters and household has no children.
+
+        # Check if filing status is separate.
+        filing_status = tax_unit("filing_status", period)
+        separate = filing_status == filing_status.possible_values.SEPARATE
+
+        # Check if tax unit has children.
+        no_children = tax_unit("tax_unit_children", period) == 0
```

Comment:
This isn't a specific definition of children, e.g. under 18 or EITC-eligible?

Reply:
Great point, just pushed change!